tf-modules/archive/db_indexer.tf

```diff
@@ -91,28 +79,6 @@ resource "aws_lambda_event_source_mapping" "granules_table_db_indexer" {
   batch_size        = 10
 }
 
-data "aws_dynamodb_table" "pdrs" {
```

Comment:
Why were these removed? I'm sure it makes sense, just curious.

Reply:
Currently, our data makes it from Dynamo to Elasticsearch via these Dynamo table streams. So every time we update a record in Dynamo, it writes that update to the table stream, which triggers the dbIndexer Lambda to write that data to Elasticsearch.

However, now that we are writing to Elasticsearch directly from the endpoint, we don't need to rely on these streams and the dbIndexer Lambda to write our data to Elasticsearch. So I'm removing these mappings for each data type as I update the endpoint logic